app/views/works/form.html.erb

```diff
@@ -53,7 +53,6 @@
           <%= render Elements::HorizontalRuleComponent.new(classes: 'my-3') %>
         <% end %>
 
-        <%= render Edit::SectionHeaderComponent.new(title: t('works.edit.panes.files.list')) %>
```

Comment:
rendering the section header is going to move into the files list component itself, since there is now some conditions on how it renders (with or without a search box next to it)